# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Improve en_DK and add en_DK.ISO-8859-15 to SUPPORTED
# DP:   This locale is only useful to provide ISO8601 date formats.
# DP: Related bugs: #323159
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Jakob Bohm <jbj@image.dk>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-16

---
 localedata/SUPPORTED     |    1 +
 localedata/locales/en_DK |   35 +++++------------------------------
 2 files changed, 6 insertions(+), 30 deletions(-)

--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -115,6 +115,7 @@
 en_CA.UTF-8/UTF-8 \
 en_CA/ISO-8859-1 \
 en_DK.UTF-8/UTF-8 \
+en_DK.ISO-8859-15/ISO-8859-15 \
 en_DK/ISO-8859-1 \
 en_GB.UTF-8/UTF-8 \
 en_GB/ISO-8859-1 \
--- a/localedata/locales/en_DK
+++ b/localedata/locales/en_DK
@@ -60,27 +60,11 @@
 END LC_CTYPE
 
 LC_MONETARY
-int_curr_symbol      "<U0044><U004B><U004B><U0020>"
-currency_symbol      "<U00A4>"
-mon_decimal_point    "<U002C>"
-mon_thousands_sep    "<U002E>"
-mon_grouping         3;3
-positive_sign        ""
-negative_sign        "<U002D>"
-int_frac_digits      2
-frac_digits          2
-p_cs_precedes        1
-p_sep_by_space       0
-n_cs_precedes        1
-n_sep_by_space       0
-p_sign_posn          1
-n_sign_posn          1
+copy "da_DK"
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        "<U002E>"
-grouping             3;3
+copy "da_DK"
 END LC_NUMERIC
 
 LC_TIME
@@ -128,8 +112,7 @@
 END LC_TIME
 
 LC_MESSAGES
-yesexpr  "<U005E><U005B><U0031><U004A><U006A><U0073><U0053><U0079><U0059><U006F><U004F><U005D><U002E><U002A>"
-noexpr   "<U005E><U005B><U0030><U006E><U004E><U005D><U002E><U002A>"
+copy "da_DK"
 END LC_MESSAGES
 
 LC_PAPER
@@ -145,17 +128,9 @@
 END LC_MEASUREMENT
 
 LC_NAME
-name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
-<U0025><U006D><U0025><U0074><U0025><U0066>"
+copy "da_DK"
 END LC_NAME
 
 LC_ADDRESS
-postal_fmt    "<U0025><U0066><U0025><U004E><U0025><U0061><U0025><U004E>/
-<U0025><U0064><U0025><U004E><U0025><U0062><U0025><U004E><U0025><U0073>/
-<U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
-<U004E><U0025><U007A><U0020><U0025><U0054><U0025>/
-<U004E><U0025><U0063><U0025><U004E>"
-country_ab2 "<U0044><U004B>"
-country_ab3 "<U0044><U004E><U004B>"
-country_num 208
+copy "da_DK"
 END LC_ADDRESS
